schwab_api/schwab.py

```diff
@@ -77,6 +77,44 @@ def get_account_info(self):
 
         return account_info
 
+    def get_transaction_history(self, account_id):
```

Comment:
Could we rename this get_transaction_history_v2(), for consistency with all other methods that are calling the "new" schwab API endpoints?

Reply:
Done!